astropy/utils/data.py

```diff
@@ -1053,17 +1070,17 @@ def _download_file_from_source(source_url, show_progress=True, timeout=None,
                         except OSError:
                             pass
                     raise
-    return f.name, hasher.hexdigest()
+    return f.name
 
 
 def download_file(remote_url, cache=False, show_progress=True, timeout=None,
-                  sources=None, pkgname='astropy', http_headers=None,
-                  ftp_tls=False):
```

Comment:
I guess the `ftp_tls` stuff made it into the LTS candidate but not a release? (So it's safe to take it back out before anyone notices?)

Reply:
Yes, as long as this is the same milestone as #9964 , we should be okay.

Reply:
Thanks for the review, @aarchiba !